samples/fbo_rtt_texture_array.html

```diff
@@ -311,6 +316,17 @@
             gl.drawArraysInstanced(gl.TRIANGLES, 0, 6, 1);
         }
 
+        console.log(w, h, w * h * 4);
+        var data = new Uint8Array( w * h * 4 * 3 );
+        gl.bindFramebuffer(gl.READ_FRAMEBUFFER, frameBuffer);
+        gl.readBuffer(gl.COLOR_ATTACHMENT0);
+        gl.readPixels(0, 0, w, h, gl.RGBA, gl.UNSIGNED_BYTE, data, 0);
+        gl.readBuffer(gl.COLOR_ATTACHMENT1);
+        gl.readPixels(0, 0, w, h, gl.RGBA, gl.UNSIGNED_BYTE, data, w * h * 4);
+        gl.readBuffer(gl.COLOR_ATTACHMENT2);
+        gl.readPixels(0, 0, w, h, gl.RGBA, gl.UNSIGNED_BYTE, data, w * h * 4 * 2);
+        console.log(data);
+        console.log( data.slice(w * h * 4 - 8, w * h * 4 + 12) );
```

Comment:
This could be a lot of logging; how about removing it? Also, since ReadPixels is expensive, may want to split its use into a separate example.